### PR TITLE
DRAFT added support for centiseconds in the TimeElementsWithFractionOfSeconds

### PR DESCRIPTION
**Pull request by rick-slin**
@joachimmetz As discussed in the plaso PR #4688, I would like to propose these change to add support for centisecond precision in TimeElementsWithFractionOfSeconds.

I think it would be easy to add 100us precision based on that template. 

Thank you for your time and consideration.

### CODE REVIEW

File: dfdatetime/time_elements.py [hunk 1]
```diff
@@ -907,7 +907,7 @@ def __init__(
 
     Args:
       fraction_of_second (Optional[decimal.Decimal]): fraction of second, which
-          must be a value between 0.0 and 1.0.
+          must be a value between 0.00 and 1.00.
```
Comment (joachimmetz) [inline dfdatetime/time_elements.py:910]: why this change?

File: tests/time_elements.py [hunk 2]
```diff
@@ -1604,5 +1605,243 @@ def testNewFromDeltaAndYear(self):
...
+        precision=definitions.PRECISION_10_MILLISECONDS,
+        time_elements_tuple=(2010, 8, 12, 20, 6, 31))
+    self.assertIsNotNone(time_elements_object)
+    self.assertEqual(
+        time_elements_object._time_elements_tuple, expected_time_elements_tuple)
+    self.assertEqual(time_elements_object.fraction_of_second, 0.87)
+
+    expected_time_elements_tuple = (2010, 8, 12, 20, 6, 31)
+    time_elements_object = time_elements.TimeElementsWithFractionOfSecond(
+      fraction_of_second=decimal.Decimal(0.8742),
```
Comment (joachimmetz) [inline tests/time_elements.py:1630]: style nit: 4 space continuation indentiation.

**Comment by joachimmetz** [comment]
@rick-slin thanks, the changes look good to me. Left a small nit and a question.

File: dfdatetime/time_elements.py (hunk 1 above)
Comment (rick-slin) [reply dfdatetime/time_elements.py:910]: I first changed it in the doc string to make it more obvious to the reader that more digits were permitted. It is pointless to have it in the code. I'll revert it.

File: dfdatetime/precisions.py [hunk 3]
```diff
@@ -99,6 +99,60 @@ def CopyToDateTimeString(cls, time_elements_tuple, fraction_of_second):
...
+  @classmethod
+  def CopyNanosecondsToFractionOfSecond(cls, nanoseconds):
+    """Copies the number of nanoseconds to a fraction of second value.
+
+    Args:
+      nanoseconds (int): number of nanoseconds.
+
+    Returns:
+      decimal.Decimal: fraction of second, which must be a value between 0.00
+      and 1.00.
```
Comment (joachimmetz) [inline dfdatetime/precisions.py:114]: style nit: missing continuation indentation

File: dfdatetime/precisions.py [hunk 4]
```diff
@@ -99,6 +99,60 @@ def CopyToDateTimeString(cls, time_elements_tuple, fraction_of_second):
...
+
+  @classmethod
+  def CopyNanosecondsToFractionOfSecond(cls, nanoseconds):
+    """Copies the number of nanoseconds to a fraction of second value.
+
+    Args:
+      nanoseconds (int): number of nanoseconds.
+
+    Returns:
+      decimal.Decimal: fraction of second, which must be a value between 0.00
```
Comment (joachimmetz) [inline dfdatetime/precisions.py:113]: no need for the addition 0 in 0.00

File: dfdatetime/time_elements.py (hunk 1 above)
Comment (joachimmetz) [reply dfdatetime/time_elements.py:910]: Noworries, I'll make the changes.

**Review by joachimmetz — APPROVED** [review]
LGTM
